holoviews/operation/datashader.py

```diff
@@ -69,6 +69,13 @@
 from ..streams import PointerXY
 from .resample import LinkableOperation, ResampleOperation2D
 
+
+def _lazy_dask_dataframe():
```

Comment:
A bit nit but I tend to think this doesn't do what people mean with lazy import (i.e. delaying importing a module until it's used). I just remembered hvPlot has a few utils that do this together with the `isinstance` check, like:

```python
def is_dask(data):
    if not check_library(data, 'dask'):
        return False
    import dask.dataframe as dd

    return isinstance(data, (dd.DataFrame, dd.Series))

def is_xarray_dataarray(data):
    if not check_library(data, 'xarray'):
        return False
    from xarray import DataArray

    return isinstance(data, DataArray)
```